composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": "^8.1",
-        "doctrine/sql-formatter": "^1.1",
+        "doctrine/sql-formatter": "^1.2",
```

Comment:
The library decided to make SQL reserved words uppercase in 1.2.

Without this bump our tests would be failing. Figure we just bump this.